iron-validator-behavior.html

```diff
@@ -52,11 +52,8 @@
      *
      * @method validate
```

Comment:
Maybe? @jklein24 suggested that annotations for methods were still a good thing to have.